src/CommunityToolkit/Xamarin.CommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -151,20 +142,21 @@
     <Reference Include="PresentationCore" />
     <Reference Include="PresentationFramework" />
     <Reference Include="WindowsBase" />
-    <PackageReference Include="Xamarin.Forms.Platform.WPF" Version="4.7.0.968" />
+    <PackageReference Include="Xamarin.Forms.Platform.WPF" Version="5.0.0.1709-pre4" />
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" />
   </ItemGroup>
 
   <ItemGroup Condition=" $(TargetFramework.StartsWith('netcoreapp')) AND '$(OS)' == 'Windows_NT' ">
     <Compile Include="**\*.wpf.cs" />
     <Compile Include="**\*.wpf.*.cs" />
-    <PackageReference Include="Xamarin.Forms.Platform.WPF" Version="4.7.0.968" />
+    <PackageReference Include="Xamarin.Forms.Platform.WPF" Version="5.0.0.1709-pre4" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" />
```

Comment:
Did we miss the package?

Reply:
Not sure what you mean by this comment?

Reply:
Pedro added a new package for this configuration. Were there any issues or we missed it when added for other platforms?

Reply:
looks like it's duplicated, probably some merge or rebase conflict. Good catch @VladislavAntonyuk